examples/src/tower/client.rs

```diff
@@ -1,8 +1,9 @@
 use hello_world::greeter_client::GreeterClient;
 use hello_world::HelloRequest;
 use service::AuthSvc;
+use tower::ServiceBuilder;
```

Comment:
should tonic re-export? Or even better could the channel build expose a way to get the servicebuilder?

Reply:
I considered that as well but opted not to. Since `ServiceBuilder` isn't very useful on its own its not clear which other things from tower we would also have to re-export.

I might consider adding some convenience stuff to channel builders but I think I'll do that as a separate PR.